radio/src/functions.cpp

```diff
@@ -376,9 +376,7 @@ void evalFunctions()
           case FUNC_SET_TIMER:
             timerSet(CFN_TIMER_INDEX(cfn), CFN_PARAM(cfn));
             break;
-#endif
```

Comment:
The CMake config is the "front end" for the user, so yes there could/should be more dummy-checking in there probably (I ran into some other situations when building different targets in random order, eg. with SDCARD or LUA being still defined on unsupported targets). I think that's a different topic.  

OTOH, if it is important to dummy-check the definitions at build-time I'd use conditional macro checks with #error/warning statements as needed, and not rely on whatever the current build system did or didn't do.  Otherwise it should just quietly build what it was told.  IMHO  :)